app/src/main/java/com/example/pinkorange/vibrato/LiveWithSettings.java

```diff
@@ -94,6 +94,7 @@ public void onCheckedChanged(CompoundButton buttonView, boolean isChecked) {
         });
 
         vibrate_seek = (SeekBar) navigationView.getMenu().getItem(4).getActionView();
+        vibrate_seek.setPadding(120, 0, 0, 0);
```

Comment:
Can we please move the padding code to the `activity_app_bar_drawer.xml ` file? I think it'll be nicer to have all of the formatting code there. What do you think?

Reply:
on it!